tigerpath/majors_and_certificates/degrees/BSE_2018.json

```diff
@@ -314,50 +314,15 @@
               "max_counted": null,
               "min_needed": null,
               "explanation": null,
-              "dist_req": "EC"
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
-              "dist_req": "EM"
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
+              "dist_req": ["EC", "EM", "HA", "LA", "SA"],
               "course_list": [
                 "LANG 1027",
                 "LANG 107",
                 "LANG 108",
                 "LANG 2**",
                 "LANG 3**",
                 "LANG 4**",
-                "LANG 5**"
-              ]
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
-              "dist_req": "HA"
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
-              "dist_req": "LA"
-            },
-            {
-              "name": null,
-              "max_counted": null,
-              "min_needed": null,
-              "explanation": null,
-              "dist_req": "SA"
+                "LANG 5**"]
```

Comment:
Just a small nit: the closing bracket should be on its own line (see the other lists in this file).